scripts/skills/perks/perk_rf_hybridization.nut

```diff
@@ -31,6 +55,31 @@ this.perk_rf_hybridization <- ::inherit("scripts/skills/skill", {
 		}
 	}
 
+	function getItemActionCost( _items )
+	{
+		foreach (item in _items)
```

Comment:
You need to check whether `this.m.IsSpent` is `true` and then return early with null.
Otherwise you can swap endlessly in a turn.